Calling `create_makasero_enhancement_issue` tool via function calling causes panic

ユーザーからの報告です。
`create_makasero_enhancement_issue` ツールを呼び出したところ、panicが発生しました。
以下にスタックトレースを記載します。

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa5f0f0]

goroutine 1 [running]:
github.com/pankona/makasero.(*Agent).processResponse(0xc0000f5540, {0xdd1a80, 0x13c7180}, 0x20?)
        /home/pankona/go/src/github.com/pankona/makasero/agent.go:245 +0x830
github.com/pankona/makasero.(*Agent).ProcessMessage(0xc0000f5540, {0xdd1a80, 0x13c7180}, {0x7ffc08c2bced, 0x277})
        /home/pankona/go/src/github.com/pankona/makasero/agent.go:166 +0x276
main.run()
        /home/pankona/go/src/github.com/pankona/makasero/cmd/makasero/main.go:137 +0xbd6
main.main()
        /home/pankona/go/src/github.com/pankona/makasero/cmd/makasero/main.go:25 +0x13

Function calling を使用してツールを実行した際に発生したようです。

--- a/functions.go
+++ b/functions.go
@@ -183,7 +183,7 @@ var builtinFunctions = map[string]FunctionDefinition{
 		},
 		Handler: handleGhIssueCreate,
 	},
-	"create_enhancement_issue": {
+	"create_makasero_enhancement_issue": {
 		Declaration: &genai.FunctionDeclaration{
 			Name:        "create_makasero_enhancement_issue",
 			Description: "makasero 自身の改善案を GitHub Issue として起票します。issue は pankona/makasero リポジトリの issue として起票され、自動的に 'enhancement' ラベルが付与されます。",

